services/graphql-server/src/graphql/definitions/platform/asset.js

```diff
@@ -71,6 +75,14 @@ input AssetImageQueryInput {
   id: ObjectID!
 }
 
+input AssetImageMutationInput {
+  id: ObjectID!
+  name: String
```

Comment:
You should probably move these fields to a `payload` child-input object (see below)